music21/midi/translate.py

```diff
@@ -520,9 +523,9 @@ def midiEventsToChord(eventList, ticksPerQuarter=None, inputM21=None):
     # of abs start time and the event object
     if isinstance(eventList, list) and isinstance(eventList[0], tuple):
```

Comment:
if isinstance(eventList, list) **and eventList** and isinstance(eventList[0], tuple):

Reply:
(this was broken before for empty MIDI files, but good to fix now